node/mock-toolrunner.ts

```diff
@@ -263,6 +263,113 @@ export class ToolRunner extends events.EventEmitter {
         });        
     }
 
+    // Deprecated, use execAsync instead
+    // Exec - use for long running tools where you need to stream live output as it runs
+    //        returns a promise with return code.
+    //
```

Comment:
Please use the `@deprecated` JSDoc tag. Otherwise this will not be shown to users in the IDE.

Reply:
Done